internal/output/table.go

```diff
@@ -283,13 +283,16 @@ func buildCostComponentRows(t table.Writer, currency string, costComponents []Co
 				c.Unit,
 			)
 
+			if c.PriceNotFound {
+				price = "not found"
+			}
```

Comment:
Changes the output table to show "not found" when component is a usage component - this is line with the non usage components.